javascripts/discourse/api-initializers/discourse_mobile_dismiss_buttons.js

```diff
@@ -5,28 +5,33 @@ import { action } from "@ember/object";
 
 export default apiInitializer("0.11.1", api => {
   api.modifyClass("component:topic-list-item", {
-    "xDown": 0,
-    "xUp": 0,
-    "yDown": 0,
-    "yUp": 0,
+    xDown: 0,
+    xUp: 0,
+    yDown: 0,
+    yUp: 0,
 
     didInsertElement() {
       this._super(...arguments);
       if (this.isTrackingTopic || this.isTrackingCategory) {
         this.addTouchListeners(this.element);
-        this.appEvents.on("mobile-swipe:untrack", this, "resetSwipe")
+        this.appEvents.on("mobile-swipe:untrack", this, "resetSwipe");
       }
     },
 
     willDestroyElement() {
       this._super(...arguments);
       this.removeTouchListeners(this.element);
-      this.appEvents.off("mobile-swipe:untrack", this, "resetSwipe")
+      this.appEvents.off("mobile-swipe:untrack", this, "resetSwipe");
     },
 
-    @discourseComputed("topic.details.notification_level")
-    isTrackingTopic(notification_level) {
-      return notification_level > 1
+    @discourseComputed(
+      "topic.notification_level",
+      "topic.details.notificaion_level"
+    )
+    isTrackingTopic(notification_level, preferredNotification_level) {
```

Comment:
You should use camel casing for there variables (`notificationLevel` and `preferredNotificationLevel`)

I think this would actually do the same thing b/c if `prefererredNotificationLevel === 0`, that is `false` in an if statement. Try it out.
```
return preferredNotificationLevel || notificationLevel
```